src/AppTypes/StaticType.php

```diff
@@ -4,4 +4,8 @@
 
 class StaticType extends BaseType
 {
+    public static function getArgs()
```

Comment:
You could do this as a default implementation on the BaseType while still making the BaseType abstract, guess it depends how common this is